### DISCUSSION TITLE
Environment Variable undefined in nuxt 2

### DISCUSSION THREAD

**Discussion by JordanQui** [vercel/community#3713] [Help] [answered]
### Summary

In my settings everything is setup correctly I have environments variables ready on Production but when I'm trying to access this variable in my app it return undefined.

Is there a configuration to be done in the Nuxt config or a command to add to the build for this version of Nuxt, so that the .env is generated correctly? 

Thank you

### Steps to Reproduce

create environment variable in a nuxt 2 deployment
try to access it with process.env.ENV_VAR in nuxt 2 app

**Comment by amyegan** [comment, marked as answer]
Hey @JordanQui. If you want to use the environment variable in a page or client-side component, then you may need to name it with the `NUXT_ENV_` prefix to indicate that it is safe to expose to the browser. You can read more about this at https://v2.nuxt.com/docs/configuration-glossary/configuration-env/#automatic-injection-of-environment-variables
